Jenkinsfile

```diff
@@ -361,21 +344,21 @@ pipeline {
           exclude {
             axis {
               name 'NODEJS_VERSION'
-              values '10'
+              values '16'
             }
             axis {
               name 'CASSANDRA_VERSION'
-              values '2.1', '4.0', 'dse-5.1.35', 'dse-6.0.18', 'dse-6.7.17'
+              values '2.1', '4.0', 'dse-5.1.35'
             }
           }
           exclude {
             axis {
               name 'NODEJS_VERSION'
-              values '12'
+              values '20'
             }
             axis {
               name 'CASSANDRA_VERSION'
-              values '2.1', '3.11', 'dse-6.0.18', 'dse-6.8.30'
+              values '2.1', '3.11', 'dse-6.8.30'
```

Comment:
The changes in this section all appear to be excludes, specifically nodejs + Cassandra versions which should be excluded from the per-commit matrix.  I'd argue we should remove this entirely for now; if we note some kind of platform-specific issue that requires exclusions along these lines we can re-introduce them but we should start by trying to run the whole matrix.

Reply:
removed the exclude section entirely